### PR TITLE
yc 646

### PR DESCRIPTION
**Pull request by monodo**

### CODE REVIEW

File: geocity/apps/accounts/templates/registration/password_reset_confirm.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+{% load static %}
+{% load i18n %}
+
+{% block content %}
+
+{% if validlink %}
+
+<h2>Geocity</h2>
+<hr>
+<h3>{% translate "Choisissez un nouveau mot de passe!" %}</h3>
```
Comment (rbovard) [inline geocity/apps/accounts/templates/registration/password_reset_confirm.html:12]:
```suggestion
<h3>{% translate "Choisissez un nouveau mot de passe" %}</h3>
```

**Review by rbovard — CHANGES_REQUESTED** [review]
I think the first commit was already merged in another PR.

I know it's outside the scope of the story, but is it possible to edit email subjet? Because the current one is not very clear `Password reset on default site`

File: geocity/apps/accounts/templates/registration/password_reset_confirm.html (hunk 1 above)
Comment (rbovard) [reply geocity/apps/accounts/templates/registration/password_reset_confirm.html]: And on the default site?

Comment (monodo) [reply geocity/apps/accounts/templates/registration/password_reset_confirm.html]: The default site should have a name...but another issue related to default site is opened

Comment (monodo) [reply geocity/apps/accounts/templates/registration/password_reset_confirm.html]: https://github.com/yverdon/geocity/issues/525

**Review by rbovard — APPROVED** [review]

**Pull request merged**